Lib/test/certdata/make_ssl_certs.py

```diff
@@ -266,6 +283,10 @@ def write_cert_reference(path):
         f.write(key)
         f.write(cert)
 
+    cert = extract_cert(cert, 0)
```

Comment:
Instead of text manipulation, could we ask openssl to extract it?
As far as I can see, the incantation is:
```
openssl x509 -outform pem -in keycert3.pem -out cert3.pem
```

Reply:
Great idea! Implemented in 021af099d48002e6dbc1405d32ebc5c54537e257.